semantics/src/main/scala/aqua/semantics/rules/ValuesAlgebra.scala

```diff
@@ -293,10 +293,11 @@ class ValuesAlgebra[S[_], Alg[_]: Monad](using
         }
     }
 
-  def valueToCallArrowRaw(v: ValueToken[S]): Alg[Option[CallArrowRaw]] =
+  def valueToCallArrowRaw(v: ValueToken[S]): Alg[Option[(ValueRaw, ArrowType)]] =
```

Comment:
nit: Lets rename it to `valueToCall` just to avoid confusion bc it returns not only `CallArrowRaw`